alf/nest/utils.py

```diff
@@ -76,7 +90,7 @@ def __init__(self, nest_mask=None, dim=-1, name="NestConcat"):
         concatenating them along a specified axis. If nest_mask is None,
         then all the tensors from the nest will be selected.
         It assumes that all the selected tensors have the same tensor spec.
-        Can be used as a preprocessing combiner in ``EncodingNetwork``.
+        Can be used as a preprocessing combiner of a ``PreprocessorNetwork``.
```

Comment:
EncodingNetwork is not a PreprocessorNetwork.

Reply:
Some preprocessor network like ActorNetwork also has combiner but they are not EncodingNetwork. So I've changed to 'network'.

Reply:
> EncodingNetwork is not a PreprocessorNetwork.

Some preprocessor network like ActorNetwork also has combiner but they are not EncodingNetwork. So I've changed to 'network'.